products/BellHybrid/services/db/ServiceDB.cpp

```diff
@@ -15,6 +18,7 @@
 ServiceDB::~ServiceDB()
 {
     eventsDB.reset();
+    multimediaFilesDB.reset();
```

Comment:
why this is needed at all?

Reply:
Removed

Reply:
Is a destructor needed at all in this case? @alekrudnik Any thoughts?